test/alphabet/nucleotide/nucl16_test.cpp

```diff
@@ -30,141 +30,233 @@
 // OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
 // DAMAGE.
 //
-// ============================================================================
-// Author: Sara Hetzel <sara.hetzel AT fu-berlin.de>
-// ============================================================================
-
-#include <array>
+// ==========================================================================
```

Comment:
the length does not match `// OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH` anymore :grin: